docs/en/common/coffee-model.adoc

```diff
@@ -36,6 +36,30 @@ public class SimpleEntity extends AbstractIdentifiedEntity {
 }
 ----
 
+We have the option to set the timezone through environment variable or system property. If we don't set this, we'll use the system's timezone by default.
+
+The variable: `COFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID`
+
+.Docker container
+[sorce,text]
+----
+service:
+    environment:
+        COFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID: Europe/Budapest
+----
+
+.Command line
+[sorce,text]
+----
+java -DCOFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID=Europe/Budapest
```

Comment:
Ez így egy kicsit ellen megy a megszokottunknak, mivel az MP Config (és majd amúgy a Quarkus configja is) úgy működik hogy van egy config kulcs, pl coffee.model.base.java.time.timezone.id , ezt keresi system properties-be, vagy egyéb propertiesben (pl wildfly.properties), és yml-be is (application.yml, microprofile-config.yml).
De ez előtt pedig az ENV-ek közt keresi, amire ezt a configkey-t átkonvertálja hogy ami . az aláhúzás lesz, és nagybetűssé teszi.

Érdemes lehet megnézni, hogy megoldható-e nálunk ez kevés befektetett idővel, hogy megmaradjon ez az egységesség még akkor is, ha ez tényleg saját változó.

Reply:
Hát az MP/Quarkus config ezt megcsinálja magától? Itt esetleg manuálisan én csinálhatnék konverziót máshogy nem tudom hogyan lehetne a felsoroltak nélkül?

Reply:
Én is erre céloztam.

Reply:
Két változóba szétszedtem őket, így jó lehet? :smile: